src/js/tabs/osd.js

```diff
@@ -1107,8 +1107,15 @@ OSD.constants = {
         GPS_RESCUE_DISABLED: {
             name: 'GPS_RESCUE_DISABLED',
             desc: 'osdWarningGpsRescueDisabled'
-        }
-
+        },
+        RSSI: {
```

Comment:
Eh, you got a bit too much now - the localised texts and the version check / adding of the warnings to `OSD.constants.WARNINGS` should stay as well - apologies for not being more explicit.

Reply:
No problem...

Reply:
Should I reference MSP 1.42 at all, or just add them back under 1.41?

Reply:
1.42 it is, as the firmware changes will go into 4.1

Reply:
Done.